swim/labels.go

```diff
@@ -145,6 +131,24 @@ func (lo LabelOptions) validateLabels(current map[string]string, additional map[
 	return nil
 }
 
+func countLabels(labels map[string]string, countPrivate bool) int {
+	if countPrivate {
+		// if we also count private labels we can simply use len()
+		return len(labels)
+	}
+
+	count := 0
+	for key := range labels {
+		if isPrivateLabel(key) {
+			// we ignore private labels in the limits
```

Comment:
You're right, this was just copied from when it was calculated in both validation functions. I'll remove it here.